lua/terrortown/entities/items/item_ttt_radar/init.lua

```diff
@@ -212,3 +211,29 @@ net.Receive("TTT2RadarUpdateAutoScan", function(_, ply)
 
 	ply.radarDoesNotRepeat = not net.ReadBool()
 end)
+
+local plymeta = assert(FindMetaTable("Player"), "FAILED TO FIND PLAYER TABLE")
+
+---
+-- Sets the radar time interval, lets the current scan run out before it is changed.
+-- @param[default=ROLE.radarTime or 30] number time The radar time interval
+-- @realm server
+function plymeta:SetRadarTime(time)
```

Comment:
I think we should avoid default values on "Setter" functions, they are confusing and unnecessary.

Reply:
but it would be way more complicated to handle the default value without this

Reply:
Where would that be then?

Reply:
I would have to provide an additionaly function like `ResetRadarTime` (or something else) to make sure there is a way to reset the radar time. Would be ok too